src/CosmosEditorManager.ts

```diff
@@ -95,7 +95,11 @@ export class CosmosEditorManager {
         const timestamp = (new Date()).toLocaleTimeString();
         ext.outputChannel.appendLine(`${timestamp}: Updated entity "${editor.label}"`);
         ext.outputChannel.show();
-        await this.updateEditor(updatedContent, vscode.window.activeTextEditor, editor);
+        if (doc.isClosed !== true) {
+            const firstRelatedEditor = vscode.window.visibleTextEditors.filter((ed) => ed.document === doc)[0];
```

Comment:
nit: Instead of `.filter()[0]`, you can use `.find()` which automatically returns the first object found.